src/test/java/org/sagebionetworks/web/unitclient/presenter/DownPresenterTest.java

```diff
@@ -107,8 +103,7 @@ public void testRepoUpHasLastPlace() {
 		presenter.checkForRepoDown();
 		verify(mockGlobalAppState).back();
 	}
-
-	@Test
+	
```

Comment:
Sorry to nitpick, I think this needs the `@Test`  annotation for it to be run.

Reply:
No problem, I added it back.